deepchecks/nlp/checks/train_test_validation/property_drift.py

```diff
@@ -139,13 +141,23 @@ def run_logic(self, context: Context) -> CheckResult:
             columns=self.properties,
             ignore_columns=self.ignore_properties
         )
+
+        # Specialized plot titles for NLP Plots
+        plot_titles = {}
+        for column_name in context.common_datasets_properties:
+            if column_name in TEXT_PROPERTIES_DESCRIPTION:
+                plot_titles[column_name] = f'{column_name}<sup><a href="{get_docs_link()}nlp/usage_guides' \
+                            '/nlp_properties.html#deepchecks-built-in-properties">&#x24D8;</a></sup><br>' \
+                            f'<sup>{TEXT_PROPERTIES_DESCRIPTION[column_name]}</sup>'
+
         results, displays = self._calculate_feature_drift(
             drift_kind='nlp-properties',
             train=train_properties,
             test=test_properties,
             train_dataframe_name=train.name or 'Train',
             test_dataframe_name=test.name or 'Test',
             common_columns=context.common_datasets_properties,
+            plot_titles = plot_titles,
```

Comment:
Maybe call this argument `override_plot_titles`?

Reply:
Done